spec/requests/notes_spec.rb

```diff
@@ -19,29 +19,67 @@
       get notes_url
       expect(response).to be_successful
     end
+
+    context 'when not signed in' do
+      before { sign_out user }
```

Comment:
But this variant reduces unnecessary code duplication, so what's the problem with this? Are our tests so slow ATM that we need to optimize them? :-)

Reply:
It's just not necessary and I think tests should as atomar as possible.  
So if a commit destroys the login system, all these specs would fail too and it would be harder to find the real problem